src/components/Button.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Button.css';
+
+const button = ({ name, width, background }) => (
+  <button
+    style={{ backgroundColor: background, width }}
+    className="button"
+    type="button"
+  >
+    {name}
+  </button>
+);
+
+button.propTypes = {
+  name: PropTypes.string.isRequired,
+  width: PropTypes.string,
+  background: PropTypes.string,
+};
+
+button.defaultProps = {
+  width: '25%',
+  background: 'grey',
```

Comment:
The default colour should be orange not grey ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)